src/components/cc-global-header.vue

```diff
@@ -124,10 +127,16 @@ export default defineComponent({
         { ID: 4, url: "#", title: "Menu 4" },
       ];
     } else {
+      vm.menuLoading = true;
       axios
         .get(requestUrl)
-        .then((response) => (vm.menus = response.data))
+        .then((response) => {
+          vm.menuLoading = false;
+          vm.errorMessage = null;
```

Comment:
I believe we can just set this to `undefined` when declaring it in the `data` object. That way, it is falsy by default and we don't need to set it to `null` here. Just one less line of code to think about.